src/util/asio_util.cpp

```diff
@@ -172,6 +172,27 @@ namespace hpx { namespace util
         return true;
     }
 
+    ///////////////////////////////////////////////////////////////////////
+    // Take an ip v4 or v6 address and "standardize" it for comparison checks
+    HPX_API_EXPORT std::string cleanup_ip_address(const std::string &addr)
```

Comment:
Export attribute are needed on the first declaration only. gcc will issue warnings for repeating it on the definition.